septima-js-application/src/main/java/com/septima/application/Data.java

```diff
@@ -19,7 +19,14 @@ private static void init(Config aConfig) {
         if (instance != null) {
             throw new IllegalStateException("Data can be initialized only once.");
         }
-        instance = new Data(new SqlEntities(aConfig.getResourcesEntitiesPath(), aConfig.getEntitiesPath(), aConfig.getDefaultDataSourceName(), Database.jdbcTasksPerformer(aConfig.getMaximumJdbcThreads()), Config.lookupExecutor()));
+        instance = new Data(new SqlEntities(
+                aConfig.getResourcesEntitiesPath(),
+                aConfig.getEntitiesPath(),
+                aConfig.getDefaultDataSourceName(),
+                Database.jdbcTasksPerformer(aConfig.getMaximumJdbcThreads()),
+                Config.lookupExecutor(),
+                Boolean.valueOf(System.getProperty("com.septima.entities.compile", "false"))

```

Comment:
Why does feature disable by default? It seems useful!

Reply:
Entities compilation is disabled by default, because runtime is the default environment.